### PR TITLE
Feature/gh 6653

### LINKED ISSUE #6653: Add "get cost centers"
This may be already implemented in "my company". If yes, we need add this into b2b checkout epic branch.

### PR DESCRIPTION
**Pull request by WeizhengSap**
close #6653

### CODE REVIEW

File: projects/core/src/user/store/reducers/user-cost-center.reducer.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import { UserActions } from '../actions/index';
+
+export const initialState: CostCenter[] = [];
+
+export function reducer(
+  state = initialState,
+  action: UserActions.UserCostCenterAction
+): CostCenter[] {
+  switch (action.type) {
+    case UserActions.LOAD_ACTIVT_COST_CENTERS_FAIL: {
```
Comment (plabadie) [inline projects/core/src/user/store/reducers/user-cost-center.reducer.ts]: Small typo in `LOAD_ACTIVT_COST_CENTERS_FAIL`

File: projects/core/src/user/store/reducers/user-cost-center.reducer.spec.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+      const { initialState } = fromUserCostCenterReducer;
+      const action = new UserActions.LoadActiveCostCentersSuccess(costCenters);
+      const state = fromUserCostCenterReducer.reducer(initialState, action);
+
+      expect(state).toEqual(costCenters);
+    });
+  });
+
+  describe('LOAD_ACTIVT_COST_CENTERS_FAIL action', () => {
```
Comment (plabadie) [inline projects/core/src/user/store/reducers/user-cost-center.reducer.spec.ts]: Small typo

File: projects/core/src/user/store/actions/user-cost-center.action.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { CostCenter } from '../../../model/org-unit.model';
+import { StateUtils } from '../../../state/utils/index';
+import { USER_COST_CENTERS } from '../user-state';
+
+export const LOAD_ACTIVE_COST_CENTERS = '[User] Load Active CostCenters';
+export const LOAD_ACTIVT_COST_CENTERS_FAIL =
```
Comment (plabadie) [inline projects/core/src/user/store/actions/user-cost-center.action.ts]: Small typo in `LOAD_ACTIVT_COST_CENTERS_FAIL`

**Review by plabadie — CHANGES_REQUESTED** [review]
There is just a small typo to fix.

File: projects/core/src/user/store/reducers/user-cost-center.reducer.ts (hunk 1 above)
Comment (WeizhengSap) [reply projects/core/src/user/store/reducers/user-cost-center.reducer.ts]: done

**Review by plabadie — APPROVED** [review]

**Pull request merged**